cpp/src/arrow/filesystem/s3fs_test.cc

```diff
@@ -166,8 +166,6 @@ Status MinioTestServer::Stop() {
 
 class S3Environment : public ::testing::Environment {
  public:
-  virtual ~S3Environment() {}
```

Comment:
I think that change is unrelated?

Reply:
It is, but i had to make it to compile with clang.

Reply:
Ah yes, we should probably include this then either here or in another PR.